index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+    <html lang="en">
+    <head>
+      <meta charset="UTF-8">
+      <meta http-equiv="X-UA-Compatible" content="IE=edge">
+      <meta name="viewport" content="width=device-width, minimum-scale=0.1"> 
+      <link rel="stylesheet" href="style.css">
+        <title> Mobile Portfolio</title>
+    </head>
+    <body>
+      <header>
+        <img class="toolbar" src="images/toolbar-mobile.png">
+      </header>
+      
+      <img class="headline" src="images/headline-mobile.png">
```

Comment:
- [ ]  Dear @supreeti Good job addressing the invalidation issues. Yet, I have some suggestions here to help you get better at changing design into a webpage.
- [ ] 1 This section of the webpage should be a `section` tag with inner contents like `main title`, `subtitle`, `paragraph`, `list of icons`, `button`. And everything should be styled according to the  [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) design. This includes `color, font, size, margin, alignment` and more. This will give your page semantic meaning and will help you customize your content in a feature. let's say you would like to change `your heading title` With your `new updated headline`, This would be hard work if we make the whole `heading` an image.